image_view/package.xml

```diff
@@ -12,6 +12,7 @@
   <maintainer email="whitleysoftwareservices@gmail.com">Joshua Whitley</maintainer>
   <maintainer email="jacob@openrobotics.org">Jacob Perron</maintainer>
   <maintainer email="mfergs7@gmail.com">Michael Ferguson</maintainer>
+  <maintainer email="siddharth.vaghela@gmail.com">Siddharth Vaghela</maintainer>
```

Comment:
> This can be reverted - maintainer is different than contributor - it's an ongoing commitment (and usually involves commit rights, reviewing PRs and releasing updates)

What is the usual process for becoming a maintainer? There isn't much information about these processes anywhere to get a clear picture of what is needed to become part of a project.

Reply:
> A lot of smaller projects don't have an official policy - generally:
> 
> 1. existing maintainers recruit other maintainers as needed, usually from people who have made consistent or significant contributions to the project over a period of time
> 
> OR
> 
> 2. if all the maintainers abandon a project, within ROS, we have some admins for the ros-* orgs that can help on board new maintainers.
> 
> Some larger projects (for instance, MoveIt) have a whole process for existing maintainers to actually vote on new promoting "core contributors" to "maintainers" (they keep a full list here: https://moveit.ros.org/about/)
> 
> I think the key to keep in mind though: ANYBODY can open a PR and work to get something into any ROS package. The response time from various maintainers obviously varies wildly. Contributing to a project is sort of a one-time commitment to get a particular PR over the threshold for acceptance, whereas maintainers are more of a longer-term commitment. With many ROS projects, you'll also find that many maintainers aren't even really actively developing on the project anymore, their contribution is in terms of 1) knowledge of the project, 2) time to review PRs, 3) periodically pushing new releases, and 4) often having access to hardware to test PRs on.

Ah, that makes sense. Thanks for sharing.